### ISSUE TITLE
Python quit unexpectedly.

### ISSUE THREAD

**Issue by P0ch1P** [MatthewPierson/Vieux#211]
Python quit unexpectedly.
I've already put IPSW in the Vieux folder. so I type "python3 Vieux -i *IPSW_NAME.ipsw*" to run.

**Screenshots**
![54f3368328acca22341a003d211f825](https://user-images.githubusercontent.com/59614108/100547785-983b1900-32a3-11eb-9685-faf2ac56faea.jpg)

Computer:
 - Computer Model [MacBook Pro 15" Early 2015]
 - MacOS Version [10.15.7/10.4.6]
 - Python Version [python 3.9 from Homebrew]
 - Vieux Version [git clone master branch at Nov 27, 2020]

**iOS Device (please complete the following information):**
 - Device: iPhone 5S 16GB A1530 [iPhone6,2]; iOS 12.4.8
 - Desired Version: iOS 10.3.3

**Additional context**
I'm fully following the Instructions; Install python3.9 by using Homebrew; Run 'pip3 install -r requirements.txt' as well.
at first, I thought it's the macOS Catalina's issue. Then I use a flash drive. format the hard drive then Downgrade to macOS Mojave. everything is a fresh installation, and still the same issue here. So frustrating.

**Comment by tirkarthi** [comment]
I have created https://github.com/MatthewPierson/Vieux/pull/212 to handle the `AttributeError` since the `fromstring` was a deprecated alias and was removed in Python 3.9 in favour of `frombytes`

**Comment by tirkarthi** [comment]
Can you please try downgrading to Python 3.8 to see if it helps?

**Comment by P0ch1P** [comment]
> Can you please try downgrading to Python 3.8 to see if it helps?

Yes, It's working. This afternoon I figure it out by myself. I use pyenv to install Python 3.7.5 instead use Homebrew to install Python directly and the iPhone Downgrade was done just by now. literally just a second ago!  Thank you anyway!
<img width="585" alt="微信图片_20201130215949" src="https://user-images.githubusercontent.com/59614108/100618897-62576c80-3357-11eb-9ca3-106189f0f3d7.png">
